internal/build/view.go

```diff
@@ -98,3 +100,13 @@ func renderBuildNumber(state string, number int) string {
 
 	return lipgloss.NewStyle().Foreground(stateColor).Render(fmt.Sprintf("Build #%d", number))
 }
+
+func trimMessage(msg string) string {
```

Comment:
There is this from bubbletea which might be better to handle more cases: https://pkg.go.dev/github.com/charmbracelet/x/ansi#Truncate

Reply:
Actually, this doesn't seem to output anything anymore? ![Screenshot 2024-06-19 at 08 29 03](https://github.com/buildkite/cli/assets/6128798/0b0bf5f7-44a9-40fc-8d8e-3ad0b1a1b5a7)

Reply:
That feels a little bit like using Bubbletea where we don't need to? We could easily add `+ "..."` to the message if we want?

Reply:
![CleanShot 2024-06-19 at 10 55 05](https://github.com/buildkite/cli/assets/10952642/e7df9aad-2350-4e9e-8a51-940b171eba6f)

Seems fine for me?

Reply:
The output with Truncate is terrible
![CleanShot 2024-06-19 at 10 59 13](https://github.com/buildkite/cli/assets/10952642/8cbe633e-d034-446b-85c4-b68085b5ce7d)

Reply:
Okay wow it seems to be a problem with my terminal (kitty). I run a pretty high line spacing and the output of that exact command is 1 line longer than a full screen. kitty seems to remove the first line from scrollback or something so that seems like a bug or configuration I'll have to look into. 

So I think this PR is good to go then